microsoft-edge/devtools-guide-chromium/customize/index.md

```diff
@@ -88,12 +88,12 @@ Navigate to [Microsoft Edge DevTools Placement][DevToolsPlacement].
    Undocked DevTools  
 :::image-end:::  
 
-## Dark theme  
+## Theme  
 
-Navigate to [Enable Dark Theme][DarkTheme].  
+Navigate to [Apply color themes to Microsoft Edge DevTools][Theme].  
 
-:::image type="complex" source="../media/customize-settings-appearance-theme.msft.png" alt-text="The dark theme" lightbox="../media/customize-settings-appearance-theme.msft.png":::
-   The dark theme  
+:::image type="complex" source="./media/customize-theme-setting.png" alt-text="Seleting a different color theme" lightbox="./media/customize-theme-setting.png":::
+   Selecting a different color theme  
 :::image-end:::  
 
```

Comment:
Possible relaxed/natural wording:
Change:
The Drawer is a second panel where tools of your choosing are displayed.
to:
The Drawer is a second panel where you can choose which tools to display.

Reply:
Per another review comment, fix typo "Seleting"